.travis.yml

```diff
@@ -7,8 +7,12 @@ php:
   - nightly
   - hhvm
 
-install:
+before_script:
   - composer self-update
-  - composer install --prefer-source
+  - composer install --dev --prefer-source
+  - mkdir -p build/logs
```

Comment:
you know composer is installed, and we are just updating it and telling it to get deps right?

I'll alter the script if it has to be done, but it being on the coveralls website, like the other website, which the recommendations of has lead to broken travis builds without changing any php source; can cause issues, which is why I was resistant to change before... Things that were working are no longer working.